sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs

```diff
@@ -134,5 +146,42 @@ public virtual Pageable<TableResponseProperties> GetTables(string select = null,
         public virtual async Task<Response> DeleteTableAsync(string tableName, CancellationToken cancellationToken = default) =>
             await _tableOperations.DeleteAsync(tableName, null, cancellationToken: cancellationToken).ConfigureAwait(false);
 
+        /// <summary> Retrieves details about any stored access policies specified on the table that may be used wit Shared Access Signatures. </summary>
+        /// <param name="table"> The name of the table. </param>
+        /// <param name="timeout"> The The timeout parameter is expressed in seconds. For more information, see <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-queue-service-operations">Setting Timeouts for Queue Service Operations.</a>. </param>
+        /// <param name="requestId"> Provides a client-generated, opaque value with a 1 KB character limit that is recorded in the analytics logs when storage analytics logging is enabled. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        [ForwardsClientCalls]
+        public virtual async Task<Response<IReadOnlyList<SignedIdentifier>>> GetAccessPolicyAsync(string table, int? timeout = null, string requestId = null, CancellationToken cancellationToken = default) =>
+            await _tableOperations.RestClient.GetAccessPolicyAsync(table, timeout, requestId, cancellationToken).ConfigureAwait(false);
```

Comment:
Any reason you are forwarding to the RestClient and not to _tableOperations directly?

Reply:
In this case, the `_tableOperations` method is just wrapping the `RestClient` call without any other modification. Perhaps it is better to be consistent and route all calls through `_tableOperations`.

Reply:
We'll soon have a change that moves diagnostic scope creation into the non-rest client and this code might be affected.